File: day1.js
const {readFileSync, promises: fsPromises} = require('fs');

filename="examples/example1.txt"

const contents = readFileSync(filename, 'utf-8').split("\n\n");

let load_per_elf = []

contents.forEach((item) => load_per_elf.push(item.split('\n').map(Number).reduce((a,b)=>a+b)))

// P1
console.log("Part one:", Math.max(...load_per_elf))

// P2

console.log(load_per_elf.sort((a,b)=>b-a).slice(0,3).reduce((a,b)=>a+b))

File: day1.py
with open('examples/example1.txt') as ifile:
    inp=ifile.read().split('\n\n')

load_per_elf=[sum([int(e) for e in elf]) for elf in [x.split() for x in inp]]

# P1
print("part 1: ", max(load_per_elf))

load_per_elf.sort()
# P2
print("part 2: ", sum(load_per_elf[-3:]))